src/xSTIR/pSTIR/STIR.py

```diff
@@ -237,6 +243,76 @@ def get_radii(self):
         ry = _float_par(self.handle, self.name, 'radius_y')
         return (rx, ry)
 
+class SingleScatterSimulator():
+    '''
+    Class for simulating the scatter contribution in an image.
+    This class
+    '''
+    def __init__(self, filename = ''):
+        self.handle = None
+        self.image = None
+        self.name = 'SingleScatterSimulation'
+        self.filename = filename
+
+        if not self.filename:
+            self.handle = pystir.cSTIR_newObject(self.name)
+        else:
+            self.handle = pystir.cSTIR_objectFromFile
+
+        check_status(self.handle)
+
+    def __del__(self):
+        if self.handle is not None:
+            pyiutild.deleteDataHandle(self.handle)
```

Comment:
This is the problematic line. Check spelling.

Reply:
Corrected. thank you